python-api/app/integrations/llm/http.py

```
@@ -0,0 +1,127 @@
+                pass
+
+        return MockResponse(data)
+
+    @staticmethod
+    def parse_json(resp: object) -> Dict[str, Any]:
+        try:
+            if hasattr(resp, "model_dump"):
+                return resp.model_dump()
+            if hasattr(resp, "dict"):
+                return resp.dict()
+            if hasattr(resp, "json") and callable(getattr(resp, "json")):
```

Ruff (0.12.2)

Do not call `getattr` with a constant attribute value. It is not any safer than normal property access.

Replace `getattr` with attribute access

(B009)